From d9a57942ec5efb72a7521510c96cdf35b10f51aa Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Fri, 23 Feb 2018 12:31:30 +0100
Subject: [PATCH] systemctl: fix indentation in output of "systemcl status" if
 there are multiple drop-in dirs

We were a few whitespace off. Let's fix that.

(cherry picked from commit eb2c3192dc6f4f4d86023f9379e1fb964a3c6c8c)
---
 src/systemctl/systemctl.c | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/src/systemctl/systemctl.c b/src/systemctl/systemctl.c
index 6ae97cf107..4b34615f2c 100644
--- a/src/systemctl/systemctl.c
+++ b/src/systemctl/systemctl.c
@@ -4063,8 +4063,9 @@ static void print_status_info(
                 char ** dropin;
 
                 STRV_FOREACH(dropin, i->dropin_paths) {
-                        if (! dir || last) {
-                                printf(dir ? "        " : "  Drop-In: ");
+                        if (!dir || last) {
+                                printf(dir ? "           " :
+                                             "  Drop-In: ");
 
                                 dir = mfree(dir);
 
@@ -4074,7 +4075,8 @@ static void print_status_info(
                                         return;
                                 }
 
-                                printf("%s\n           %s", dir,
+                                printf("%s\n"
+                                       "           %s", dir,
                                        special_glyph(TREE_RIGHT));
                         }
 
-- 
2.17.0

